libs/leonsans/src/leonsans.ts

```diff
@@ -0,0 +1,608 @@
+  }
+
+  get breakWord() {
+    return this.breakWord_;
+  }
+
+  get isPattern() {
+    return this.isPattern_;
+  }
+
+  set isPattern(v) {
+    this.isPattern_ = v;
+    this.updatePatternPaths(true);
+  }
+
+  get isWave() {
+    return this.isWave_;
+  }
+
+  set isWave(v) {
+    this.isWave_ = v;
+    this.updateWavePaths(true);
+  }
+
+  get hasGrid() {
+    return this.hasGrid_;
+  }
+
+  set hasGrid(v) {
+    this.hasGrid_ = v;
+  }
+
+  get hasPoint() {
+    return this.hasPoint_;
+  }
+
+  set hasPoint(v) {
+    this.hasPoint_ = v;
+  }
+
+  get hasBox() {
+    return this.hasBox_;
+  }
+
+  set hasBox(v) {
+    this.hasBox_ = v;
+  }
+
+  get fps() {
+    return this.fps_;
+  }
+
+  set fps(v) {
+    this.fps_ = v;
+    this.frameInterval_ = 1000 / this.fps_;
+  }
+
+  get lineWidth() {
+    return this.model.lineWidth;
+  }
+
+  get scale() {
+    return this.model.scale;
+  }
+
+  get drawing() {
+    return this.model.drawing;
+  }
+
+  get data() {
+    return this.model.data;
+  }
+
+  /**
+   * 리온 산스의 위치를 설정합니다.
+   * @param x x 좌표
+   * @param y y 좌표
+   */
+  position(x = 0, y = 0) {
+    if (this.model.position(x, y)) {
+      this.updateId_++;
+      this.updateSignal();
+    }
+  }
+
+  update() {
+    this.updateId_++;
+
+    this.model.update(
+      this.str_,
+      this.width_,
+      this.breakWord_,
+      this.weight_,
+      this.size_,
+      this.tracking_,
+      this.leading_,
+    );
+
+    if (this.isPattern_ || this.isWave_) {
+      this.updatePatternPaths();
+      this.updateWavePaths();
+    } else {
+      this.updateSignal();
+    }
+  }
+
+  updateGuide() {
+    if (this.guideId_ != this.updateId_) {
+      this.guideId_ = this.updateId_;
+      this.model.updateGuide();
+    }
+  }
+
+  /**
+   * Update paths for drawing in WebGL (PIXI.js). It's very expensive, only call when it needs.
+   */
+  updateDrawingPaths() {
+    if (this.drawingPathsId_ != this.updateId_) {
+      this.drawingPathsId_ = this.updateId_;
+      this.model.updateDrawingPaths();
+    }
+  }
+
+  /**
+   * Update paths for pattern
+   * @param {boolean} force - Force execution
+   */
+  updatePatternPaths(force?: boolean) {
+    if (this.isPattern_ && (force || this.patternPathsId_ != this.updateId_)) {
+      this.patternPathsId_ = this.updateId_;
+      this.model.updatePatternPaths(this.pathGap_);
+      this.isForceRender_ = true;
+      this.updateSignal();
+    }
+  }
+
+  /**
+   * Update paths for wave effect
+   * @param {boolean} force - Force execution
+   */
+  updateWavePaths(force?: boolean) {
+    if (this.isWave_ && (force || this.wavePathsId_ != this.updateId_)) {
+      this.wavePathsId_ = this.updateId_;
+      this.model.updateWavePaths(this.pathGap_);
+      this.isForceRender_ = true;
+      this.updateSignal();
+    }
+  }
+
+  updateSignal() {
+    this.model.updateLinesForRect();
+    this.model.updatePathsForRect();
+    this.dispatch('update', this.model);
+  }
+
+  reset() {
+    this.size_ = 500;
```

Comment:
```suggestion
    this.size_ = DEFAULT_FONT_SIZE;
```
아님 적어도 사이즈 값은 constants에서 가져와야 할 거 같음